.github/workflows/functional-validation-publish.yml

```diff
@@ -0,0 +1,50 @@
+name: FunctionalValidation-PublishPackage
+
+on:
+  pull_request:
+    branches: 
+      - develop
+    paths:
+      - '.github/workflows/functional-validation-publish.yml'
+      - 'src/shared/FunctionalValidation/**'
+    types: [closed]
+
+     
+jobs:
+  publish:
+    if: ${{ github.event.pull_request.merged }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 6.0.x
```

Comment:
Can we keep it as GitHub var? I think there would be more `dotnet-version: x.y.z`

Reply:
I've created local GitHub action that loads variables from env files in `.github/vars` directory.